test/js-native-api/test_string/test_string.c

```diff
@@ -120,6 +120,22 @@ static napi_status create_external_latin1(napi_env env,
   return napi_ok;
 }
 
+static napi_status create_property_key_utf16(napi_env env,
```

Comment:
This function is not used and can be removed.

Reply:
I removed it thank you very much